website/src/js/components/SearchTipsModal.js

```diff
@@ -93,10 +107,35 @@ bar to refine your search. You can change how new terms or phrases relate to you
                 <img src={MetadataFilters} alt='Metadata Filters Screenshot' />
               </div>
           </div>
-          <div className="row" id="LogicalOperators">
+          <div className="row" id="Scoring">
             <div className="col-8">
               <SearchTipsModalItem
                 number='3'
+                header='Scoring search results'
```

Comment:
@parksjr This is _my_ only change to this PR. If I understand the comments in the JIRA ticket correctly I think the rest are made by you and Ron.